packages/react-components/src/components/abstract-output-component.tsx

```diff
@@ -75,7 +76,7 @@ export abstract class AbstractOutputComponent<P extends AbstractOutputProps, S e
             </div>
             <div className='main-output-container' ref={this.mainAreaContainer}
                 style={{ width: this.props.widthWPBugWorkaround - this.props.style.handleWidth, height: this.props.style.height }}>
-                {this.renderMainArea()}
+                {this.checkAndRenderMainArea()}
```

Comment:
I would suggest renderMainOutputContainer(), see line 77 className. I would perhaps also rename the ref variable to mainOutputContainer, to be consistent.

Reply:
Great suggestion! It feels clean and consistent.